thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/notification/content/templates/HierarchicalAnomaliesContent.java

```diff
@@ -201,4 +211,38 @@ private AnomalyReportEntity putAnomaliesIntoRootOrLeaf(MergedAnomalyResultDTO an
     }
     return anomalyReport;
   }
+
+  /**
+   * Retrieve the average wow baseline values
+   * @param anomaly an instance of MergedAnomalyResultDTO
+   * @param compareMode the way to compare, WoW, Wo2W, Wo3W, and Wo4W
+   * @param start the start time of the monitoring window in millis
+   * @param end the end time of the monitoring window in millis
+   * @return baseline values based on compareMode
+   * @throws Exception
+   */
+  private Double getAvgComparisonBaseline(MergedAnomalyResultDTO anomaly, COMPARE_MODE compareMode,
+      long start, long end) throws Exception{
+    AnomalyFunctionFactory anomalyFunctionFactory = new AnomalyFunctionFactory(thirdEyeAnomalyConfig.getFunctionConfigPath());
```

Comment:
`AnomalyFunctionDTO ` is no longer in use.  We can use the baseline provider to get the baseline numbers instead.

Reply:
I want to refrain from making any functionality changes in this PR. Moreover this unused template will require some separate refactoring to make it work with the new pipeline. So I'd postpone that for now.